pkgs/development/libraries/hamlib/4.nix

```diff
@@ -22,11 +22,11 @@ let
 in
 stdenv.mkDerivation rec {
   pname = "hamlib";
-  version = "4.5.5";
+  version = "4.6";
 
   src = fetchurl {
     url = "mirror://sourceforge/${pname}/${pname}-${version}.tar.gz";
```

Comment:
```suggestion
    url = "mirror://sourceforge/hamlib/hamlib-${version}.tar.gz";
```

Reply:
@SuperSandro2000 Thanks for the review. I've included all your suggested changes.. 
..looking forward to your re-review...